frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/common.embedding.unit.spec.tsx

```diff
@@ -101,6 +140,10 @@ describe("[OSS] embedding settings", () => {
             /Interactive embedding with full, granular control./,
           ),
         ).toBeInTheDocument();
+        expect(
+          withinEmbeddingSdkCard.getByLabelText("Disabled"),
+        ).not.toBeChecked();
+        expect(withinEmbeddingSdkCard.getByLabelText("Disabled")).toBeEnabled();
```

Comment:
`toBeEnabled` is to ensure that the toggle is always enabled (clickable). `not.toBeChecked()` (`enable-embedding-sdk: false`) asserts that the value we passed to `enable-embedding-sdk` is read and displayed in the toggle correctly.

I will try it out if `not.toBeChecked()`/`toBeChecked()` assumes `toBeEnabled()` or not. I doubt it does anyway, since you could have a disabled input with values inside.

There are other tests that assert similarly but asserts that `toBeChecked()` (`enable-embedding-sdk: true`).

There are also tests that assert that the CORS origin input is `toBeDisabled` when certain conditions are met.

Reply:
@WiNloSt ohhhh, I mistakenly read "toBeEnabled" as "if the feature is enabled" and I was misled big time. It make sense now that it's "if the checkbox itself is enabled"

My bad, I just need more sleep 😆